src/components/layout/CardTitle.js

```diff
@@ -0,0 +1,33 @@
+import classnames from 'classnames';
+
+import { downcastRef } from '../../util/typing';
+
+/**
+ * @typedef {import('../../types').PresentationalProps} CommonProps
+ * @typedef {import('preact').JSX.HTMLAttributes<HTMLDivElement>} HTMLDivAttributes
+ *
+ */
+
+/**
+ * Style a title for a Card
+ *
+ * @param {CommonProps & HTMLDivAttributes} props
+ */
+export default function CardTitle({
```

Comment:
I wonder if we should be using a semantic heading of some sort for card titles?

Reply:
Good idea. I'll address this when I address adding `icon` to `CardHeader` in a follow-up.